src/core/query/metadata.pl

```diff
@@ -23,7 +23,7 @@
 transaction_object_triple_count(Transaction, Triple_Count) :-
     transaction_object_read_write_objs(Transaction,Objects),
     maplist(read_object_triple_count,Objects,Counts),
-    sumlist(Counts,Triple_Count).
+    sum_list(Counts,Triple_Count).
```

Comment:
Why is this change necessary? Are these predicates containing `sumlist` not being used?

Reply:
It couldn't autoload sumlist but I could not find the library that it would need to be imported from. Therefore I changed it to the non-deprecated one since the predicate signature is the same.